include/enums/enum_error_estimator_type.h

```diff
@@ -40,7 +40,8 @@ enum ErrorEstimatorType : int {
                          LAPLACIAN               =  5,
                          PATCH_RECOVERY          =  6,
                          WEIGHTED_PATCH_RECOVERY =  7,
-                         UNIFORM_REFINEMENT      =  8};
+                         UNIFORM_REFINEMENT      =  8,
+                         SMOOTHNESS              =  9};
```

Comment:
Calling the SmoothnessEstimator an ErrorEstimator feels like a category error to me.  We're returning a smoothness, which should be non-zero for the exact solution, rather than error, which shouldn't be, right?